### PR TITLE
Prepare MySQL datastore for quickstart

### PR DESCRIPTION
**Pull request by khanhtc1202**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
Enable to use MySQL datastore for quickstart
```

### CODE REVIEW

File: manifests/pipecd/templates/deployment.yaml [hunk 1]
```diff
@@ -252,36 +252,42 @@ spec:
...
-        - name: mongodb
-          image: mongo:{{ .Values.mongodb.imageTag }}
+        - name: mysql
+          image: mysql:{{ .Values.mysql.imageTag }}
           imagePullPolicy: IfNotPresent
+          env:
+            - name: MYSQL_ROOT_PASSWORD
+              value: test
+            - name: MYSQL_DATABASE
+              value: quickstart
```
Comment (nghialv) [inline manifests/pipecd/templates/deployment.yaml:283]: Those values should be specified in the `values.yaml` file. (Same with `imageTag`)

Comment (khanhtc1202) [reply manifests/pipecd/templates/deployment.yaml]: Addressed on https://github.com/pipe-cd/pipe/pull/1742/commits/8ad14197a34cdc761165ea9b0fe0fe2d18263240 🙏

File: manifests/pipecd/values.yaml [hunk 2]
```diff
@@ -38,9 +38,11 @@ ops:
     logEncoding: humanize
   resources: {}
 
-mongodb:
-  imageTag: "4.2"
+mysql:
+  imageTag: "8.0.23"
   resources: {}
+  rootPassword: "test"
+  database: "quickstart"
```
Comment (nghialv) [inline manifests/pipecd/values.yaml:45]: Sorry, my bad. Please move these values to `quickstart/control-plane-values.yaml`.

Comment (khanhtc1202) [reply manifests/pipecd/values.yaml]: 確かにw 👍 
https://github.com/pipe-cd/pipe/pull/1742/commits/548d4b5c046cc5ab1c19c5c8086a7f8b757919ef 🙏

**Comment by nghialv** [comment]
Thank you.
/approve

**Pull request merged**